.env.dist

```diff
@@ -6,6 +6,8 @@ VUE_APP_PRIMARY_COLOR=#015d99
 VUE_APP_SECONDARY_COLOR=#791f5e
 VUE_APP_ANALYTICS_ID='UA-xxxxxxxxx-y'
 VUE_APP_RECAPTCHA_ID=''
+VUE_APP_SENTRY_DSN=123
+
```

Comment:
I guess this would have to be added to the skinny repos. Doesn't each have there own in Sentry?